### PR TITLE
Add instructions to get the latest apk from the artifacts

### LINKED ISSUE #1055: Add Instructions to get the latest apk after github actions in README
After migrating to Github actions, we can get the latest apk files from artifacts. We need to add instructions for the same in the README file.

### PR DESCRIPTION
**Pull request by Sparsh1212**
## Issue Fix

Fixes #1055 

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Sparsh1212** [comment]
@devansh-299 @shiv07tiwari Please review this.

File: README.md [hunk 1]
```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
 * `gradlew spotlessApply` an check and apply formatting to any plain-text file.</br>
 * `gradlew build`  provides a command line to execute build script.</br>
 
+To get the latest apk fom the Github actions artifacts, follow these steps
```
Comment (s-ayush2903) [inline README.md]: Make this an eye catching title via making it some h3/h2/bold with lesser than 7 words, as it is very difficult to find this procedure in the readme for downloading

File: README.md [hunk 2]
```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
...
+
+1. Navigate to the **Actions** tab of this repository.
```
Comment (s-ayush2903) [inline README.md]: I think it would be easier for people to follow if _Actions_ is hyperlinked : )

File: README.md [hunk 3]
```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
...
+2. Click on your particular workflow from the workflows list.
+3. Scroll down to the **Artifacts** section and click the **mobile-wallet** hyperlink.
+4. The zip file download will start.
```
Comment (s-ayush2903) [inline README.md]: I don't think this is a step that _needs to be done manually_, maybe we can get rid of it.

File: README.md [hunk 4]
```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
...
+5. After successful download, extract the zip file to your preferred location.
+6. After extracting the zip file, you will get the **mifospay-debug.apk** in your selected location.
```
Comment (s-ayush2903) [inline README.md]: Same as the trailing comment.

**Review by s-ayush2903 — COMMENTED** [review]
Good work, but it can better if we consider a few subtle changes

File: README.md (hunk 3 above)
Comment (Sparsh1212) [reply README.md:102]: @s-ayush2903 Can you please specify which step no. specifically among these.

Comment (Sparsh1212) [reply README.md:102]: > I don't think this is a step that _needs to be done manually_, maybe we can get rid of it.

@s-ayush2903 Can you please specify which step no. specifically among these.

File: README.md (hunk 1 above)
Comment (Sparsh1212) [reply README.md:99]: > Make this an eye catching title via making it some h3/h2/bold with lesser than 7 words, as it is very difficult to find this procedure in the readme for downloading

@s-ayush2903 Hpw about this ?

File: README.md [hunk 5]
```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
 * `gradlew spotlessApply` an check and apply formatting to any plain-text file.</br>
 * `gradlew build`  provides a command line to execute build script.</br>
 
+### Instructions to get the latest apk
```
Comment (s-ayush2903) [inline README.md]:
```suggestion
### Instructions to get the latest APK
```

File: README.md [hunk 6]
```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
...
+
+To get the latest apk fom the Github actions artifacts, follow these steps
```
Comment (s-ayush2903) [inline README.md]:
```suggestion
To get the latest apk fom the Github actions artifacts, follow these steps:
```

**Review by s-ayush2903 — COMMENTED** [review]
Ah a few more minute changes needed

File: README.md [hunk 7]
```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
...
+
+1. Navigate to the [Actions](https://github.com/openMF/mobile-wallet/actions) tab of this repository.
+2. Click on your particular workflow from the workflows list.
```
Comment (s-ayush2903) [inline README.md]: Umm, you know what, on following this one will get the apk generated for the lastest triggered workflow(that can be of any pull request which isn't merged in the dev) and the core idea of the corresponding issue is to make people able to download the latest stable apk, which would always be generated via latest workflow triggered on the `dev` branch. 

Let me know if some clarification is needed.

Comment (s-ayush2903) [reply README.md]: Don't prioritize this, I'm not very sure of the idea. I'll leave it upto @devansh-299 regarding what apk we want people to download, the one corresponding to their Pull Request(which they can obtain by clicking the workflows from their PR directly) or the latest stable one.

Comment (Sparsh1212) [reply README.md]: > Don't prioritize this, I'm not very sure of the idea. I'll leave it upto @devansh-299 regarding what apk we want people to download, the one corresponding to their Pull Request(which they can obtain by clicking the workflows from their PR directly) or the latest stable one.

@s-ayush2903 Sure! I'll do this if he says so. Also, thanks a lot for the review.

Comment (devansh-299) [reply README.md]: > 
> 
> Don't prioritize this, I'm not very sure of the idea. I'll leave it upto @devansh-299 regarding what apk we want people to download, the one corresponding to their Pull Request(which they can obtain by clicking the workflows from their PR directly) or the latest stable one.

The main idea of building APKs and saving them as artifacts was to allow people to test out the latest stable build without having the need of installing Android Studio on their systems. The contributors already have the environment set up to build their projects and test it, hence it makes sense to link the actions for merged changes. For doing this you can simply query the `Event` as _push_

Comment (s-ayush2903) [reply README.md]: Okay, so @Sparsh1212, so update accordingly i.e., specifying workflow query to the `push` event to the `dev` branch

**Comment by Sparsh1212** [comment]
@s-ayush2903 @devansh-299 Please review the latest changes.

File: README.md [hunk 8]
```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
 * `gradlew spotlessApply` an check and apply formatting to any plain-text file.</br>
 * `gradlew build`  provides a command line to execute build script.</br>
 
+### Instructions to get the latest APK
+
+To get the latest apk fom the Github actions artifacts, follow these steps:
+
+1. Navigate to the [Actions](https://github.com/openMF/mobile-wallet/actions?query=workflow%3A%22Mobile-Wallet+CI%5BMaster%2FDev%5D%22+event%3Apush) tab of this repository.
+2. Click the particular workflow from the workflows list.
```
Comment (s-ayush2903) [inline README.md]: As discussed earlier, we want the _latest_ apk to be downloaded

```suggestion
2. Click the latest workflow from the workflows list.
```

**Comment by Sparsh1212** [comment]
@devansh-299 Please review. Also, shall I squash all the commits into a single commit now ?

**Comment by devansh-299** [comment]
> 
> 
> @devansh-299 Please review. Also, shall I squash all the commits into a single commit now ?

Looks good to me, also please squash the commits.

**Comment by Sparsh1212** [comment]
> > @devansh-299 Please review. Also, shall I squash all the commits into a single commit now ?
> 
> Looks good to me, also please squash the commits.

@devansh-299 Done!

**Pull request merged**